chat-microservice/app/core/config.py

```diff
@@ -1,9 +1,11 @@
-from typing import Any, Dict, Optional
+import os
 
+from typing import Any, Dict, Optional
```

Comment:
Again, no need to import `Dict` on 3.9
```suggestion
from typing import Any, Optional
```